lib/scotty.js

```diff
@@ -8,27 +8,29 @@ const reuseBucket = require('./reuse-bucket')
 const upload = require('./upload')
 const getFormattedSize = require('./get-formatted-size')
 const setAsWebsite = require('./set-as-website')
+const createRoute = require('./create-route')
 const setCdn = require('./set-cdn')
 
 let time = new Date()
 
-function createOrUpdateBucket(bucket, region, update, force, quiet, logger) {
-  if (update)
-    return Promise.resolve(bucket)
+function createOrUpdateBucket(options) {
```

Comment:
Using options is cleaner, but not sure if I like it because it's less readable and it requires additional checks. But let's keep it for now.